app/controllers/application_controller.rb

```diff
@@ -13,7 +13,18 @@ def after_sign_in_path_for(resource)
   private
 
   # we can enable this for everybody, it's hidden by default
-  def enable_rack_mini_profiler
-    Rack::MiniProfiler.authorize_request
+  def rack_mini_profiler
+    show =
+      if params.key?(:enable_rack_mini_profiler)
+        cookies[:rack_mini_profiler] = "show"
+        true
+      elsif params.key?(:disable_rack_mini_profiler)
+        cookies.delete(:rack_mini_profiler)
+        false
+      end
```

Comment:
@arielj I don't want to be a blocker, you are free to pick this suggestion.

I'm seen that we are adding `enable_rack_mini_profiler` and `disable_rack_mini_profiler` but I think it will be a little bit better if we use it as `/projects?rack_mini_profile=true` or `/projects?rack_mini_profile=false`.
that way we have consistence in the argument and will be easy to just change `true/false` to enable/disabled.

Reply:
hmmm I wasn't sure about this, should we only support `true`/`false`? what about `1`/`0`? or `True`/`False`? or `enable`/`disable`? or `enabled`/`disabled`?

I think that calling the query string param `enable_rack_mini_profiler` and `disable_rack_mini_profiler` is more expressive and explicit, you either enable or disable rack_mini_profiler.

Using `rack_mini_profiler=true` or `=false` leaves me with some questions, what does it do? it enables it? for how long? why is it `true`/`false` and not `enable`/`disable`? I feel it's more `developer-ish` but not that developer friendly.